pyproject.toml

```diff
@@ -18,6 +18,9 @@ classifiers = [
 "Bug Tracker" = "https://github.com/TJStienstra/symmeplot/issues"
 "Documentation" = "https://tjstienstra.github.io/symmeplot/"
 
+[tool.ruff]
+show-source = true
```

Comment:
You might want to think about adding additional configuration. For example, only the "E" and "F" rules are enabled by default. The ["A"](https://beta.ruff.rs/docs/rules/#flake8-builtins-a) and ["W"](https://beta.ruff.rs/docs/rules/#warning-w) rules are also important, the ["I"](https://beta.ruff.rs/docs/rules/#isort-i) rules replace isort, and the ["D"](https://beta.ruff.rs/docs/rules/#pydocstyle-d) for pydocstyle.

Reply:
Thanks noticed it and I'm currently looking into which I can enable. `"D"` requires some docstyle changes, so messing a bit with sphinx again.